src/main/java/org/wildfly/security/auth/spi/AuthenticatedRealmIdentity.java

```diff
@@ -35,6 +36,13 @@
     Principal getPrincipal();
 
     /**
+     * Returns the roles for this identity.
+     *
+     * @return A string set containing the roles for this identity or an empty set if this identity has no roles.
```

Comment:
Hi @jaikiran :)  I created them using reveal.js with a slightly modified version of one of the default themes.  The source code is at https://github.com/dmlloyd/presentations/tree/gh-pages/coding-standards.